chef/cookbooks/neutron/recipes/common_config.rb

```diff
@@ -185,56 +180,61 @@
 num_vlans = neutron[:neutron][:num_vlans]
 vlan_end = [vlan_start + num_vlans - 1, 4094].min
 
+
 case neutron[:neutron][:networking_plugin]
```

Comment:
Just thinking out loud (so this should not block this PR, just some idea for cleanup): If I am not mistaken, everything from here down to the ```end``` is either related to the l2 agent configuration or to the configuration of the nsx plugin. Wouldn't it make sense to move the l2-agent related bits to ```common_agent``` and the nsx plugin related things to ```server.rb```?

That way the bits would only get executed where they are really needed.